.vscode/launch.json

```diff
@@ -20,6 +20,7 @@
       "env": {
         "FLASK_APP": "${workspaceRoot}/web_app.py",
         "FLASK_ENV": "local",
+        "ENV_PREFIX": "DC",
```

Comment:
curious why this needs to be set?

Reply:
This was a recent change made to the script: https://github.com/datacommonsorg/website/blob/9b3df3ee1b94d9c10d036da75672d342bf9896f1/run_server.sh#L62

But it breaks ability to run server in vscode for debugging, and this change fixes it.